components/base/sensorcontrolled/sensorcontrolled.go

```diff
@@ -71,6 +72,11 @@ type sensorBase struct {
 	allSensors  []movementsensor.MovementSensor
 	orientation movementsensor.MovementSensor
 	velocities  movementsensor.MovementSensor
+
+	lvctrl              *linearVelControllable
```

Comment:
I think writing out the words instead of "lv" and "av" would be better (or at least "linVel" and "angVel"). Later on in the file the variables don't feel clear at a glance

Reply:
since this is a draft pull request I'm resolving the comments on names and styling. Will revisit once we're at that stage.